### PR TITLE
Fixes #12775: Patch the agent Makefile in rudder-packages in order to build it on Slackware

### PR DESCRIPTION
**Pull request by VictorQrt**
https://www.rudder-project.org/redmine/issues/12775

### CODE REVIEW

**Comment by VictorQrt** [comment]
Commit modified

**Comment by VictorQrt** [comment]
Commit modified

File: rudder-agent/SOURCES/Makefile [hunk 1]
```diff
@@ -176,6 +176,11 @@ ifneq (false,$(USE_ACL))
 ACL_ARG := --with-libacl
 endif
 
+# In case we do not want pam
```
Comment (peckpeck) [inline rudder-agent/SOURCES/Makefile]: Please write the default in the comment

**Comment by VictorQrt** [comment]
Commit modified

**Pull request merged**